src/main/java/racingcar/model/domain/Cars.java

```diff
@@ -0,0 +1,24 @@
+package racingcar.model.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class Cars {
+
+    private List<Car> cars;
+
+    public Cars(final List<String> userInput) {
+        run(userInput);
+    }
+
+    public List<Car> cars() {
+        return Collections.unmodifiableList(cars);
+    }
+
+    public void run(final List<String> input) {
+        cars = new ArrayList<>();
+        input.stream().forEach(carNmae -> cars.add(new Car(carNmae)));
```

Comment:
Collection에서의 forEach는 Stream을 쓰지않아도 괜찮아요~!

```suggestion
        input.forEach(carName -> cars.add(new Car(carName)));
```

Reply:
로직이 하나 인데 불필요하게 stream을 사용한듯 싶습니다. forEach로 반영하였습니다.